src/odemis/driver/test/tescan_test.py

```diff
@@ -49,8 +49,8 @@
 # However, not everything behaves exactly as on the real hardware, so beware
 
 # arguments used for the creation of basic components
-CONFIG_SED = {"name": "sed", "role": "sed", "channel": 0, "detector": "e-t"}
-CONFIG_FIB = {"name": "sed", "role": "sed", "channel": 0, "detector": "se"}
+CONFIG_SED = {"name": "sed", "role": "sed", "channel": 0}
+CONFIG_FIB = {"name": "sed", "role": "sed", "channel": 0}
```

Comment:
Can you add a test case that changes the detector type? (Just pick one detector from the choices, which is different from the current value, set it, and acquire an image)